tests/test_playwright_real_gemini_e2e.py

```
@@ -0,0 +1,458 @@
+                    # h1が見つからない場合、すべてのタイトル要素を確認
+                    all_title_elements = await page.query_selector_all("title, h1, h2, h3, .title, .header")
+                    print(f"タイトル関連要素の数: {len(all_title_elements)}")
+                    
+                    if all_title_elements:
+                        for i, elem in enumerate(all_title_elements):
+                            text = await elem.text_content()
+                            tag = await elem.evaluate("el => el.tagName")
+                            print(f"タイトル要素{i}: {tag} - {text}")
+                    
+                    # h1が見つからない場合はエラーとする
+                    assert False, "h1タグが見つかりません"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)